.github/workflows/container-version.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron: '37 13 * * *'
+name: busybox-update workflow
+jobs:
+  checkVersionAndCreatePR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run busybox updater
+        run: |
+          chmod +x ./scripts/busybox-updater.sh
+          ./scripts/busybox-updater.sh
+        shell: bash
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          signoff: true
```

Comment:
Maybe we can add `branch: update-sha-action` and `delete-branch: true` here so that the branches are auto-deleted after merge and we know what branches are made by this action?

Reply:
It creates a patch, so I believe there is less of a mess in PR style. We can further check things by the PR's it's making as reference if needed. If this behaviour needs to be changed we can iterate on it. 
Basically it's now in the style of dependabot.

Reply:
implemented